graphql-java-support/src/test/resources/com/apollographql/federation/graphqljava/schemas/product.graphql

```diff
@@ -8,7 +8,11 @@ type Money {
 }
 
 type Product @key(fields : "upc") {
+  """
+  The product "name".
+  """
```

Comment:
Would it make sense to include test cases for all characters that require block strings here?